astropy_bot/autolabel.py

```diff
@@ -72,6 +72,9 @@ def autolabel(pr_handler, repo_handler):
                                 installation=pr_handler.installation)
 
     al_config = upstream_repo.get_config_value("autolabel", {})
+    if not al_config.get("enabled", True):
```

Comment:
In the other PR, you do this ```enabled = false``` for the config. Would that not be a `False` here, given that `'false'` would parse as a string? 

Which is all OK, I think it's more like an issue for baldrick. If I'm not totally mistaken by this, then anyway, the max we can do here is to document that disabling can be done by commenting out the config item in the configfile.

Reply:
Re: "false" vs False -- I am following https://github.com/OpenAstronomy/baldrick/blob/3008bb1cc00adc5826774e0e71847633f0901fa0/baldrick/plugins/github_pull_requests.py#L110 + https://github.com/astropy/astropy-bot/blob/007711d2dbaeada02c13439b8963dcb0704a0512/pyproject.toml#L9 unless they are wrong too? Maybe @Cadair or @astrofrog can advise.

If you do not want to disable it, you do not have to add anything to `pyproject.toml` because the default here is that it is enabled. But I'll add a note on the astropy PR side.

Reply:
Hmm, but if my point is true (which I'm not sure about, there cannot be such a big bug around), then there is no way to switch this off.

Anyway, one way to test it is to merge this, and see how the astropy PR reacts after the deploy. If it's a bug, then it should be fixed not just here, so it would be beyond the scope of this PR anyway.